pages.fr/common/wc.md

```diff
@@ -0,0 +1,20 @@
+# wc
+
+> Compte les mots, les octets ou les lignes.
+
+- Compte les lignes d'un fichier:
+
+`wc -l {{file}}`
+
+- Compte les motes d'un fichier:
+
+`wc -w {{file}}`
+
+- Count les caractères (octets) d'un fichier:
```

Comment:
```suggestion
- Compte les caractères (octets) d'un fichier:
```
Is this a typo?

Other than this, looks fine :+1:

Reply:
Indeed that is a typo. It has been fixed in the next commit.